README.md

```diff
@@ -126,6 +126,10 @@ Similar to `no_focus_stealing_in`. If a tag enable this, then the tag will stay
 selected no matter what event happen. This is useful for video games and video
 players.
 
+**no_focus_only:**
```

Comment:
The name is a bit confusing, what do you think about `no_tag_deselect`?

Reply:
yeah, I wasn't so sure about the name either. `no_tag_deselect` sounds reasonable.

Reply:
oops, sorry, it's called `awful.tag.viewmore(c:tags())`, I should fix that, it's confusing